src/docs/codelabs/layout-basics.md

```diff
@@ -1,427 +1,646 @@
+
+<ul markdown="1">
+  <li markdown="1">
+  An `Icon` widget set to `Icons.account_circle` and with a size of 50 pixels.
+
+  </li>
+  <li markdown="1">
+  A `Padding` widget that creates a space of 8 pixels around the `Icon` widget. 
+
+  To do this, you can specify `const EdgeInsets.all(8.0)` for the `padding` property.
+
+  The `Row` should look like this: 
+  </li>
+</ul>
+
+  ```dart
+     Row( 
+      children: [
+        Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Icon(Icons.account_circle, size: 50),
+        ),
+     Column( ... ), // <--- The Column you first implemented
+      ],
+     );   
+  ```
+{{site.alert.end}}
+{% comment %}
+  Gist: https://gist.github.com/datafoya/95dcc1451aea8412669c41eb8a1a5f23
+{% endcomment %}
+<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=95dcc1451aea8412669c41eb8a1a5f23&amp;theme=dark&amp;split=60" width="100%" height="400px"></iframe>
 
-Your first `Row` is now complete! There are two more to go, though,
-and you need a `Column` to put them in.
-Wrap your `Row` with a `Column` widget so that it looks like this:
+### Part 2
+{:.no_toc}
 
-<!-- skip -->
-```dart
- Column(
-   children: [
-     Row( … ), // <- This should be the Row with your Icon and Text widgets.
-   ],
- );
-```
+#### Exercise: Tweak the layout
+{:.no_toc}
+{{site.alert.secondary}}
+  
+  Wrap the `Row` in a `Column` that has a `mainAxisSize` property set to `MainAxisSize.min`
+  and a `crossAxisAlignment` property set to `CrossAxisAlignment.stretch`. 
+  The `Column` contains the following widgets:
 
-Then, finish up your new `Column` with these steps:
+  * A `SizedBox` widget with a height of 8.
 
-* Set the `Column`'s `mainAxisSize` to min
-  * Otherwise it'll expand to fill the screen!
+  * An empty `Row` where you'll add the contact information. 
 
-* Set the `Column`'s `crossAxisAlignment` to stretch
-  * This makes all of its children full-width
+  * A second `SizedBox` widget with a height of 16.
 
-* Add more widgets below your `Row` in the `Column`'s
-  list of children:
-  * A `SizedBox` with a height of 8
-  * An empty `Row` (no children or other properties)
-  * A `SizedBox` with a height of 16
-  * Another empty `Row`
+  * A second empty `Row` where you'll add different the four icons (Part 3).
 
-<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=19ead6db4f42ce112fc0a7d2e0922466&split=60" width="100%" height="800px"></iframe>
+  The `Column`'s list of widgets should be formatted like this, 
+  so the contact information and icons are displayed below the name and title: 
 
-There are just a few steps to go now. Next up is the second row.
-Add the following to its list of children:
+  ```dart
 
-* A `Text` widget with a street address like '123 Main Street'
-* A `Text` widget with a phone number like '800-123-1234'
+     ],
+    ), // <--- Closing parentheses for the Row
+    SizedBox(),
```

Comment:
I still find it a little confusing to see "SizedBox()" instead of "SizedBox(...)" since it otherwise seems exactly like the code I wrote. Not a huge deal, but gotta let you know.